vue-frontend/src/components/about/HowItAllStartedSection.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="3xl:tw-container !tw-mt-20 tw-mb-8 xl:tw-mb-20 2xl:tw-mb-48">
+    <h1
+      class="tw-text-center tw-text-[1.875rem] md:tw-text-[2.813rem] tw-leading-[2.25rem] md:tw-leadig-[3.875rem] tw-font-inter-bold tw-text-black-core/[0.87]"
+    >
+      How it all started?
+    </h1>
+    <div class="tw-flex tw-flex-row tw-pt-10 xl:tw-pt-20">
+      <div class="tw-w-[60%] tw-overflow-hidden tw-py-40">
+        <div
+          class="tw-relative tw-mt-[-120px] 2xl:tw-mt-[-33px] 3xl:tw-mt-[-100px]"
+        >
+          <ul
+            ref="years"
+            :style="{
+              left: `${left}px`,
+            }"
+            class="tw-ml-[20%] tw-w-[80%] 3xl:tw-w-full tw-absolute tw-transition-all tw-ease-in-out tw-duration-1000 tw-flex tw-flex-nowrap tw-justify-between tw-w-full tw-list-none"
+          >
+            <li
+              v-for="(story, index) in stories"
+              :key="story"
+              :ref="'year-' + index"
+              :class="
+                index == activeIndex
+                  ? 'v2-canopas-gradient-text'
+                  : index < activeIndex
+                  ? 'tw-text-black-core/[0.2]'
+                  : 'tw-text-black-core/[0.6]'
+              "
+              class="tw-flex tw-justify-center tw-text-[1.5rem] tw-leading-[2.5rem] xl:tw-text-[1.875rem] xl:tw-leading-[2.812rem] tw-font-inter-medium hover:tw-bg-clip-text hover:tw-text-transparent hover:tw-from-[#FF835B] hover:tw-to-[#F2709C] hover:tw-bg-gradient-[270.11deg]"
+              @click="
+                animate = true;
+                slide(index, story);
```

Comment:
Let's add all the functionalities in slide() method only instead of listing here